config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "prefix": "nom-",
```

Comment:
Are we good with this prefix? I keep typing `npm` by accident :(

Reply:
Looks good to me

Reply:
what does nom stand for?

Reply:
Nom nom nom, since this bot is a play on a pig foot.